core/src/toga/style/pack.py

```diff
@@ -259,23 +261,44 @@ def __delitem__(self, name):
     # End backwards compatibility
     ######################################################################
 
-    def apply(self, prop: str, value: object) -> None:
+    def apply(self, name: str, value: object = NOT_PROVIDED) -> None:
+        ######################################################################
+        # 2025-02: Backwards compatibility for Toga <= 0.4.8
```

Comment:
```suggestion
        # 2025-02: Backwards compatibility for Toga < 0.5
```

Reply:
Actually... there's some other comments in the same space that need to be updated in the same way. I'll piggyback those onto this PR.